subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/UpToDateScalaCompileIntegrationTest.groovy

```diff
@@ -16,13 +16,17 @@
 
 package org.gradle.scala.compile
 
+
```

Comment:
Nit: Extra newline. I think IDEA does this sometimes when refactoring. There may be even a bug open.

Reply:
Done